cdap-api/src/main/java/co/cask/cdap/api/workflow/WorkflowActionEntry.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.api.workflow;
+
+/**
+ * Represents entry for the {@link WorkflowAction}
+ */
+public final class WorkflowActionEntry {
```

Comment:
Renamed the class to ProgramNameTypeInfo. This class simply maintains the name of the program and corresponding SchedulableProgram type.